go/ngql/query_test.go

```diff
@@ -35,7 +35,7 @@ func (suite *QueryGraphQLSuite) assertQueryResult(v types.Value, q, expect strin
 
 func (suite *QueryGraphQLSuite) TestScalars() {
 	suite.assertQueryResult(types.String("aaa"), "{root}", `{"data":{"root":"aaa"}}`)
-	// suite.assertQueryResult(types.String(""), "{root}", `{"data":{"root":""}}`)
+	suite.assertQueryResult(types.String(""), "{root}", `{"data":{"root":""}}`)
```

Comment:
Note that I patched our vendered copy of graphql-go and referenced an existing bug.